src/client/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,27 @@
+
+import * as React from "react";
+
+import './Navbar.less';
+
+
+export default class Navbar extends React.Component {
```

Comment:
@Evilart86 I think you should divide this component to smaller components,
NavbarContainer, NavbarList, NavbarActions - something like that